tests/functional/adapter/test_utils.py

```diff
@@ -648,7 +673,6 @@ def models(self):
 
 -- Also test correct casting of literal values.
 
-union all select {{ datediff("'1999-12-31 23:59:59.999999'", "'2000-01-01 00:00:00.000000'", "microsecond") }} as actual, 1 as expected
```

Comment:
Are we removing the support for microsecond?

Reply:
Yaa removed this hacky piece of code too: https://github.com/cloudera/dbt-impala/pull/196/files/5a31621c4e583939adb861501f9ca5b78fdbaa49

Reply:
can we remove it as a separate commit?